themes/radial/components/gantt-chart.typ

```diff
@@ -0,0 +1,135 @@
+#import "../colors.typ": *
+#import "../icons/icons.typ": *
+#import "@preview/timeliney:0.0.1"
+
+/// Creates a gantt chart for task management.
+///
+/// Example Usage:
+///
+/// ```typ
+/// #gantt_chart(
+///   months: (
+///     (month: [January], length: 4),
+///     (month: [February], length: 4),
+///   ),
+///   dates: (
+///     (date : [Week 1], length: 1),
+///     (date : [Week 2], length: 1),
+///     (date : [Week 3], length: 1),
+///     (date : [Week 4], length: 1),
+///     (date : [Week 1], length: 1),
+///     (date : [Week 2], length: 1),
+///     (date : [Week 3], length: 1),
+///     (date : [Week 4], length: 1),
+///   ),
+///   tasks: (
```

Comment:
This part of the API is fine in my opinion. The only criticism I have is to maybe make the color parameter optional, and have a set list of colors that get chosen from if the parameter isn't selected.